guides/source/debugging_rails_applications.md

```diff
@@ -461,11 +490,11 @@ the `thread` command (or the abbreviated `th`). This command has a handful of
 options:
 
 * `thread`: shows the current thread.
-* `thread list`: is used to list all threads and their statuses. The plus +
-character and the number indicates the current thread of execution.
-* `thread stop _n_`: stop thread _n_.
-* `thread resume _n_`: resumes thread _n_.
-* `thread switch _n_`: switches the current thread context to _n_.
+* `thread list`: is used to list all threads and their statuses. The current
+thread is marked with a plus (+) sign.
+* `thread stop n`: stops thread _n_.
+* `thread resume n`: resumes thread _n_.
+* `thread switch n`: switches the current thread context to _n_.
```

Comment:
> We don't follow that.

Sorry, I thought you did (not as in "we'll introduce major edits so all our guides follow it", but as in "it's nice to have, you can change it as you change other stuff"). I thought so because I actually formatted the guide to follow it (eb2d555f5c2e504f9de73dd083f148336f5ffa19). Maybe your criteria changed?

> Yup, asking you for a lot of trouble, but we will need to change everywhere else :smile: .Thanks!

What are the changes you want me to do? Currently everywhere where "\_n\_" is used inside backquotes is fixed to be just "n".